ydb/core/protos/kqp.proto

```diff
@@ -391,6 +391,7 @@ message TEvCreateSessionRequest {
     optional string ClientSdkBuildInfo = 10;
     optional string ApplicationName = 11;
     optional string UserSID = 12;
+    optional string UserName = 13;
```

Comment:
Maybe it is better to make some king of PgSessionInfo message or send serialized GUCSettings?